doc/Dashboard.md

```diff
@@ -0,0 +1,91 @@
+"Dashboard" Self-Assessment Summary Block
+=========================================
+
+A "Dashboard" XBlock provides a concise way to summarize a student's answers to
+groups of multiple choice questions.
+
+You configure it like this, by pasting in the `url_name`s of some Problem
+Builder ("Mentoring") blocks:
+![Screen shot of Dashboard XBlock configuration](img/dashboard-configuration.png)
+
+
+And it will then look like this (after a student has submitted their answers):
+![Screen shot of a Dashboard XBlock](img/dashboard-example.png)
```

Comment:
![screen shot 2015-03-27 at 1 31 14 pm](https://cloud.githubusercontent.com/assets/1099483/6874485/9b74591a-d485-11e4-87ba-84d90aadc648.png)


Mine looks more like this. Looks like it's capturing the value instead of the label, which seems unintuitive to me.